changelogs/unreleased/gh-9159-fix-memleak-in-sql-colunm-default-parser.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/sql
+
+* Fixed a memory leak in column default parser (gh-9159).
```

Comment:
```suggestion
* Fixed a memory leak in a column default parser (gh-9159).
```

Reply:
We will merge this commit in the scope of PR https://github.com/tarantool/tarantool/pull/9214. This commit here is only for testing.